packages/go/analysis/impact/id_aggregator.go

```diff
@@ -114,7 +114,7 @@ func (s IDA) pushDependency(target, dependency uint32) {
 
 // popDependencies will take the simplex cardinality provider reference for the given target, remove it from the
 // containing map in the aggregator and then return it
-func (s IDA) popDependencies(targetUint32ID uint32) []uint32 {
+func (s IDA) popDependencies(targetUint32ID uint64) []uint64 {
```

Comment:
nit: this param name and a few others within this file as well as the var name in the AddShortcut method use uint32 nomenclature which could be confusing even though everything is functional. We should update the param and var names to match the actual type used

Reply:
Updated.